packages/image_picker/ios/Classes/ImagePickerPlugin.m

```diff
@@ -251,6 +251,13 @@ - (void)imagePickerController:(UIImagePickerController *)picker
 
     NSNumber *maxWidth = [_arguments objectForKey:@"maxWidth"];
     NSNumber *maxHeight = [_arguments objectForKey:@"maxHeight"];
+    NSNumber *imageQuality = [_arguments objectForKey:@"imageQuality"];
+
```

Comment:
Maybe add 
```objc
if (![ImageQuality isKindOfClass:[NSNumber class]]) {
   imageQuality = @1;
} 
```

This made the imageQuality an optional parameter and returns 1 to be default. Same on Android

Reply:
done